src/content/dvt/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Distributed Validator Technology
+description: Distributed Validator Technology enables the distributed operation of an Ethereum validator by multiple parties.
+lang: en
+---
+
+# Distributed Validator Technology {#what-is-dvt}
+_With the advent of Ethreum’s PoS consensus, standard validator setups were accompanied by some challenges and the risk of stake centralization. To help distribute validator nodes and decentralize the network, Distributed Validator Technology was conceived._
```

Comment:
```suggestion
```
This relies on a lot of prior knowledge - lets just jump in to the start of the next section and ease the reader in more gently.